package/verification-template/gradle/verification-metadata.xml

```diff
@@ -534,6 +560,22 @@
             <sha256 value="1da5fdcd12e75b62ba85eed94ab46b64ab9ad7f66a5d1cfcbe194e4e9a0f5d84" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="dev.koifysh" name="archipelago-client" version="0.1.18">
+         <artifact name="archipelago-client-0.1.18.jar">
+            <sha256 value="8ebb3dbaa85dd6edebd348151ead99b781f27fddacaef33a8d5f516c21db5ade" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="archipelago-client-0.1.18.pom">
+            <sha256 value="3354696605f14bc952380651786ab3bc97a4bbd5a63d74a5973f177c18c17725" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
```

Comment:
is this necessary given https://github.com/digiholic/osrs-archipelago/blob/34a7bacdda632ae961998562a3982647de17fd5d/build.gradle#L28